dev/fattest.simplicity/src/componenttest/rules/repeater/MicroProfileActions.java

```diff
@@ -70,208 +67,276 @@ public class MicroProfileActions {
-                                                          "mpRestClient-1.4" };
+                                                          "mpRestClient-1.4",
+                                                          "mpContextPropagation-1.0", //standalone
+                                                          "mpGraphQL-1.0", //standalone
+                                                          "mpLRA-1.0", //standalone
+                                                          "mpLRACoordinator-1.0", //standalone
+                                                          "mpReactiveMessaging-1.0", //standalone
+                                                          "mpReactiveStreams-1.0" }; //standalone
 
     private static final String[] MP40_FEATURES_ARRAY = { "microProfile-4.0",
-                                                          "servlet-4.0",
-                                                          "cdi-2.0",
-                                                          "jaxrs-2.1",
-                                                          "jaxrsClient-2.1",
-                                                          "jsonb-1.0",
-                                                          "jsonp-1.1",
+                                                          "servlet-4.0", //ee8
+                                                          "cdi-2.0", //ee8
+                                                          "jaxrs-2.1", //ee8
+                                                          "jaxrsClient-2.1", //ee8
+                                                          "jsonb-1.0", //ee8
+                                                          "jsonp-1.1", //ee8
                                                           "mpConfig-2.0",
                                                           "mpFaultTolerance-3.0",
                                                           "mpHealth-3.0",
                                                           "mpJwt-1.2",
                                                           "mpMetrics-3.0",
                                                           "mpOpenAPI-2.0",
                                                           "mpOpenTracing-2.0",
-                                                          "mpRestClient-2.0" };
+                                                          "mpRestClient-2.0",
+                                                          "mpContextPropagation-1.2", //standalone
+                                                          "mpGraphQL-1.0", //standalone
+                                                          "mpLRA-1.0", //standalone
+                                                          "mpLRACoordinator-1.0", //standalone
+                                                          "mpReactiveMessaging-1.0", //standalone
+                                                          "mpReactiveStreams-1.0" }; //standalone
```

Comment:
`mpReactiveMessaging-1.0` is not compatible with `mpConfig-2.0`. I would remove both reactive features from MP 4.0 and 4.1.